app/services/container_dashboard_service.rb

```diff
@@ -119,29 +121,44 @@ def get_url_to_entity(entity)
 
   def heatmaps
     # Get latest hourly rollup for each node.
-    node_ids = @ems.container_nodes if @ems
+    node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.includes(:resource => [:ext_management_system]) unless @ems.present?
+
+    node_cpu_usage = nil
+    node_memory_usage = nil
```

Comment:
@zeari what would be the problem to set these to arrays here? (What's effect in the UI? Is it different from nil?)